blocks/carousel/carousel.css

```
@@ -270,4 +299,19 @@ main .section div.carousel-wrapper {
         text-align: left;
         margin: 0;
     }
+
+    .carousel.image-carousel-full-width .carousel-slide img {
+        width: 120px;
+        height: 120px;
+    }
+
+    .carousel-container .carousel.image-carousel-full-width .carousel-slide-container {
+        gap: 108px;
+        padding: 0 var(--spacer-element-10);
+    }
+
+    .carousel.image-carousel-full-width.no-animation-effect .carousel-slide-container {
+        grid-template-columns: 120px 120px 120px 120px;
+        column-gap: 108px;
```

```suggestion
        column-gap: var(--gap-108);
```
Let's move this value to our CSS variable for managing gaps. https://github.com/hlxsites/merative2/blob/main/styles/styles.css#L206

---

Done